contrib/test_jenkins.sh

```diff
@@ -755,18 +793,37 @@ run_ucx_perftest_mpi() {
 		if [[ $ucx_dev =~ .*mlx5.* ]]; then
 			opt_transports="-b $ucx_inst_ptest/transports"
 			tls=`awk '{print $3 }' $ucx_inst_ptest/transports | tr '\n' ',' | sed -r 's/,$//; s/mlx5/x/g'`
-			ucx_env_vars="-x UCX_NET_DEVICES=$ucx_dev -x UCX_TLS=$tls"
+			dev=$ucx_dev
 		elif [[ $ucx_dev =~ posix ]]; then
 			opt_transports="-x mm"
-			ucx_env_vars="-x UCX_TLS=mm"
+			tls="mm"
+			dev="all"
+		elif [[ " ${ip_ifaces[*]} " == *" ${ucx_dev} "* ]]; then
+			opt_transports="-x tcp"
+			tls="tcp"
+			dev=$ucx_dev
 		else
 			opt_transports="-x rc"
-			ucx_env_vars="-x UCX_NET_DEVICES=$ucx_dev -x UCX_TLS=rc"
+			tls="rc"
+			dev=$ucx_dev
 		fi
 
 		echo "==== Running ucx_perf kit on $ucx_dev ===="
-		$MPIRUN -np 2 $AFFINITY $UCT_PERFTEST -d $ucx_dev $opt_transports
-		$MPIRUN -np 2 $ucx_env_vars $AFFINITY $UCP_PERFTEST
+		if [ $with_mpi -eq 1 ]
+		then
+			$MPIRUN -np 2 $AFFINITY $UCT_PERFTEST -d $ucx_dev $opt_transports
+			$MPIRUN -np 2 -x UCX_NET_DEVICES=$dev -x UCX_TLS=$tls $AFFINITY $UCP_PERFTEST
+		else
+			export UCX_NET_DEVICES=$dev
+			export UCX_TLS=$tls
+
+			run_client_server_app "$UCT_PERFTEST" "-P 0 -d ${ucx_dev} ${opt_transports}" \
```

Comment:
1. maybe better to pass test args (e.g batch file) as "test_args" instead of adding it to "UCT_PERFTEST" variable
e.g there will be $UCX_PERFTEST, but $uct_test_args and $ucp_test_args
1. What is "-P 0" argument?
1. Need to add CPU affinity according to executor number with "-c" param